django/db/models/indexes.py

```diff
@@ -44,21 +44,27 @@ def check_name(self):
             self.name = 'D%s' % self.name[1:]
         return errors
 
-    def create_sql(self, model, schema_editor, using=''):
+    def index_parameters(self, model, schema_editor, using=''):
```

Comment:
Suggested name: `get_sql_create_params`. `BrinIndex` should override this function rather than overriding `create_sql`.